.ci/e2e-expected/select-dml-batching.txt

```diff
@@ -0,0 +1 @@
+ERROR:  INVALID_ARGUMENT: The statement is not suitable for batching. Only DML and DDL statements are allowed for batching.
```

Comment:
print the argument i.e. the exact statement that is causing the error

Reply:
I added the sql string so the user can see what caused the problem.